api4/api.go

```diff
@@ -47,6 +47,8 @@ type Routes struct {
 	ChannelsForTeam          *mux.Router // 'api/v4/teams/{team_id:[A-Za-z0-9]+}/channels'
 	ChannelMembers           *mux.Router // 'api/v4/channels/{channel_id:[A-Za-z0-9]+}/members'
 	ChannelMember            *mux.Router // 'api/v4/channels/{channel_id:[A-Za-z0-9]+}/members/{user_id:[A-Za-z0-9]+}'
+	ChannelPreviewers		 *mux.Router // 'api/v4/channels/{channel_id:[A-Za-z0-9]+}/previewers'
+	ChannelPreviewer 		 *mux.Router // 'api/v4/channels/{channel_id:[A-Za-z0-9]+}/previewer/{user_id:[A-Za-z0-9]+}'
```

Comment:
The api path should be plural, in keeping with convention 

```suggestion
	ChannelPreviewer 		 *mux.Router // 'api/v4/channels/{channel_id:[A-Za-z0-9]+}/previewers/{user_id:[A-Za-z0-9]+}'
```